apps/cowswap-frontend/src/legacy/state/orders/orderPrice.test.ts

```
@@ -23,14 +33,114 @@ describe('getEstimatedExecutionPrice()', () => {
+      surplusAmount: 2 * 10 ** WETH_MAINNET.decimals,
+    }
+
+    const remainderAmounts = getRemainderAmountsWithoutSurplus(order)
+
+    expect(remainderAmounts.buyAmount).toEqual('4112319564256476036857852')
+    expect(remainderAmounts.sellAmount).toEqual('4542619072256475291189243')
+  })
+
+  it.skip('should return the remainder amounts if there is surplus when passing a SELL Order', () => {
+    const order = getLimitOrderWithFee(
+      17.42619072256475129773952e23,
```

Biome (1.9.4)

[error] This number literal will lose precision at runtime.

The value at runtime will be 1742619072256475000000000

(lint/correctness/noPrecisionLoss)